general/management/commands/announcement_cache.py

```diff
@@ -0,0 +1,45 @@
+"""Custom management command to set/clear announcement_cache for front page announcements banner.
+
+The announcement banner is a simple HTML snippet cached in the Django cache. It is formed by a title and a text
+that are passed as arguments to the command. The command can be used to set the cache, clear it, or show the current
+value of the cache.
+
+Example usage:
+    python manage.py announcement_cache set "New feature!" "Now you can do this and that. <a href='/forum'>Learn more</a>"
+    python manage.py announcement_cache clear
+    python manage.py announcement_cache show
+
+"""
+from django.core.cache import cache
+from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
+
+
+class Command(BaseCommand):
+    help = 'Set/clear "announcement_cache" for front page announcements banner'
+
+    def add_arguments(self, parser):
+        parser.add_argument("action", type=str, help="Indicates whether to set or clear the cache")
+        parser.add_argument("title", type=str, nargs="?", help="Title of the announcement")
+        parser.add_argument("text", type=str, nargs="?", help="Text of the announcement")
+
+    def handle(self, *args, **kwargs):
+        action = kwargs["action"]
+        title = kwargs["title"]
+        text = kwargs["text"]
+
+        announcement_cache_key = "announcement_cache"
+        if action == "set":
+            one_day = 60 * 60 * 24
```

Comment:
The number of days could be a parameter of the command as well, and I'd say that if not specified, then it could have no expiration

Reply:
is now an optional argument to the command